examples/image_classification.ipynb

```diff
@@ -0,0 +1,5278 @@
+{
```

Comment:
<p>I think you can get rid of the last sentence....it's not really saying it "in other words" 😅</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>